requirements.txt

```diff
@@ -1,4 +1,5 @@
-numpy
-scipy
+numpy>=1.7
```

Comment:
As mentioned in #440, let's drop these requirements for now.

Reply:
Feel free @MuawizChaudhary to open an issue!